ui/src/main/webapp/src/app/components/reports/migration-issues/migration-issues-table.component.ts

```diff
@@ -79,4 +83,13 @@ export class MigrationIssuesTableComponent implements OnInit {
 
         return this.problemSummariesFiles.get(issue);
     }
+
+    navigateToSource(file:any) {
+        let fileModel = <FileModel>new GraphJSONToModelService().fromJSON(file, this._http, FileModel);
```

Comment:
I'm wondering if we really want to make HTTP requests directly in components? I think we usually do that in services.